src/Symfony/Component/DependencyInjection/Dumper/PhpDumper.php

```diff
@@ -1725,6 +1726,11 @@ private function dumpValue(mixed $value, bool $interpolate = true): string
 
             try {
                 if ($value instanceof ServiceClosureArgument) {
+                    $attributes = '';
+                    if ($value->getName()) {
+                        $attributes = sprintf('#[\%s(name: "%s")] ', ClosureAttribute::class, $value->getName());
```

Comment:
we should use `$this->dumpValue()` to generate the dumped name

Reply:
Done